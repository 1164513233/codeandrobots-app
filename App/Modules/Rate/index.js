
import Rate from './Screens/Containers'

export {
  Rate
}
