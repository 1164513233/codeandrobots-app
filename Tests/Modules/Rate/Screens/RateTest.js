import React from 'react'

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Rate from 'App/Modules/Rate/Screens/Components'

describe('Rate', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(
      <Rate onRatePress={() => {}} />)
  })

  it('Component renders correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
