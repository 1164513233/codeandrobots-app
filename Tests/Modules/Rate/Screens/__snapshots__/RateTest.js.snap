// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rate Component renders correctly 1`] = `
<Container>
  <Footer>
    <Card
      button="Rate"
      onPress={[Function]}
      text="Every rating means the world to us.

★★★★★"
      title="LOVING THE APP?"
    />
  </Footer>
</Container>
`;
